cmd/devp2p/internal/ethtest/suite.go

```diff
@@ -398,3 +399,28 @@ func (s *Suite) dial() (*Conn, error) {
 
 	return &conn, nil
 }
+
+func (s *Suite) TestTransaction(t *utesting.T) {
+	tests := []*types.Transaction{
+		getNextTxFromChain(t, s),
+		unknownTx(t, s),
+	}
+	for i, tx := range tests {
+		fmt.Printf("Testing tx propagation: %v\n", i)
```

Comment:
This message, and other things printed to stdout, will be lost when running the tests in hive. Please use `t.Logf` instead.